x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -410,6 +410,15 @@ export function LayerPanel(
                     )}
                   </>
                 }
+                labelAppend={
+                  isOptional ? (
+                    <span className="lnsLayerPanel__rowOptional">
+                      {i18n.translate('xpack.lens.editorFrame.optionalDimensionLabel', {
+                        defaultMessage: 'Optional',
+                      })}
+                    </span>
```

Comment:
It may be better to utilize an `EuiText` component here, rather than a custom styled `span`. Thoughts?

```suggestion
                    <EuiText className="lnsLayerPanel__rowOptional" color="subdued" size="xs">
                      {i18n.translate('xpack.lens.editorFrame.optionalDimensionLabel', {
                        defaultMessage: 'Optional',
                      })}
                    </EuiText>
```

Reply:
Great, thank you!